assets/prefabs/Equipment/Gold/GoldStrengthnLuck.prefab

```diff
@@ -0,0 +1,29 @@
+"parent" : "engine:iconItem",
```

Comment:
FYI you somehow lost the leading `{` here. No big deal so I fixed it real quick :-)